src/app/teams/team-edit/team-edit.component.html

```diff
@@ -0,0 +1,89 @@
+<md-toolbar>
+    <app-back-button
+        #backButton
+    ></app-back-button>
+    <app-title></app-title>
+    <div class="toolbar-icon-spacer"></div>
+</md-toolbar>
+
+<form [formGroup]="form">
+    <md-card>
+        <div class="edit-input-line">
+            <md-input-container [dividerColor]="form.controls.info.valid ? 'primary' : 'warn'" #infoContainer>
+                <textarea mdInput
+                          #info
+                          [style.width]="'100%'"
+                          type='text'
+                          formControlName="info"
+                          placeholder="Enter team info">
+                    <md-hint *ngIf="infoContainer._mdInputChild.focused">
+                        Describe your team for potential players
+                    </md-hint>
+                    <md-hint *ngIf="infoContainer._mdInputChild.focused" align="end">
+                        {{ info.value.length }} / 1000
+                    </md-hint>
+                </textarea>
+            </md-input-container>
+            <div class="edit-input-button">
+                <button md-icon-button (click)="save()" *ngIf="form.controls.info.value != sourceTeam.info">
+                    <md-icon>save</md-icon>
+                </button>
+                <div class="mat-icon-button" *ngIf="form.controls.info.value == sourceTeam.info"></div>
+            </div>
+        </div>
+        <div class="edit-input-line">
+            <md-radio-group formControlName="type">
+                <md-radio-button value="2">Men</md-radio-button>
+                <md-radio-button value="1">Women</md-radio-button>
+                <md-radio-button value="0">Coed</md-radio-button>
+            </md-radio-group>
+            <div class="edit-input-button">
+                <button md-icon-button (click)="save()" *ngIf="form.controls.type.value != sourceTeam.type">
+                    <md-icon>save</md-icon>
+                </button>
+                <div class="mat-icon-button" *ngIf="form.controls.type.value == sourceTeam.type"></div>
+            </div>
+        </div>
+    </md-card>
+
+    <hr/>
+    <md-card>
+        <div class="inline-roaster-header">
+            <h1>Roaster</h1>
+            <button md-mini-fab *ngIf="isPlayer || isManager" [routerLink]="['/players', {targetTeam: team.id}]"><md-icon>add</md-icon></button>
```

Comment:
Some lines in this file are way too long, they don't even fit in the github viewer